### PR TITLE
CODENVY-426 show dedicated message when clone fails due to lack of SSH key

### PR DESCRIPTION
**Pull request by mshaposhnik**

### CODE REVIEW

**Comment by TylerJewell** [comment]
@riuvshin If we accept this request, we should cherry pick this issue and codenvy-455 into a hot fix release.  Maybe combined with the swarm race condition fix.

**Comment by mshaposhnik** [comment]
@skabashnyuk  @vparfonov pls check

File: plugins/plugin-bitbucket/codenvy-plugin-bitbucket-ext-bitbucket/src/main/java/org/eclipse/che/ide/ext/bitbucket/server/BitbucketKeyUploader.java [hunk 1]
```diff
@@ -1,3 +1,4 @@
+
```
Comment (sleshchenko) [inline plugins/plugin-bitbucket/codenvy-plugin-bitbucket-ext-bitbucket/src/main/java/org/eclipse/che/ide/ext/bitbucket/server/BitbucketKeyUploader.java]: remove this line

**Comment by skabashnyuk** [comment]
ок

**Pull request merged**